pkgs/games/prismlauncher/default.nix

```diff
@@ -38,8 +38,8 @@ rec {
   src = fetchFromGitHub {
     owner = "PrismLauncher";
     repo = "PrismLauncher";
-    rev = version;
-    sha256 = "sha256-RArg60S91YKp1Mt97a5JNfBEOf2cmuX4pK3VAx2WfqM=";
+    rev = "caf925cbad6cbd2f09ec2e55526b072b7958c5d5";
```

Comment:
I don't think there is a need to bump to a development version. If users are interested in development versions, they should just use the flake `github:PrismLauncher/PrismLauncher#prismlauncher`

Reply:
That was an accident, I will fix.